lib/exercises.rb

```diff
@@ -1,19 +1,45 @@
 
 # This method will return an array of arrays.
 # Each subarray will have strings which are anagrams of each other
-# Time Complexity: ?
-# Space Complexity: ?
+# Time Complexity: O(m + n + nlog(n))
+# Space Complexity: O(n)
 
 def grouped_anagrams(strings)
```

Comment:
👍 I would say this is n * m log m where n is the number of strings and m is the length of the strings.  Or if the strings are all less than a certain size, O(n) for time complexity.

Otherwise well done.